package.json

```diff
@@ -50,24 +53,34 @@
     "preset": "jest-react-native"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.9",
     "@expo/webpack-config": "^0.10.4",
+    "@types/jest": "^26.0.19",
     "@types/react": "^16.8.6",
-    "@types/react-native": "^0.60.0",
-    "babel-jest": "16.0.0",
+    "@types/react-native": "^0.63.37",
+    "@types/react-test-renderer": "^17.0.0",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.6.3",
     "babel-preset-expo": "^7.1.0",
+    "eslint": "^7.15.0",
```

Comment:
All of these are needed after your changes in babel and tsconfig? @jakub-gonet

Reply:
- I added `eslint-plugin-import` & `eslint-import-resolver-babel-module` which is needed to make Eslint aware of babel aliases,
- `metro-react-native-babel-preset` because it was used in config but we relied on the fact that some package would bring it in, 
- `prettier`, `babel-jest`, `@types/react-native`, `react-native` are bumped,
- some more types are added too

So everything is needed (or I missed something)